examples/grpc-server/message.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package llama;
+
+option go_package = "./pkg/grpc";
+
+service LlamaGoService {
+    rpc Answer(Job) returns (stream Output){}

```

Comment:
I suggest following aip.dev style design here, I'd expect this to be something more like `rpc PerformCompletion(PerformCompletionRequest) returns (PerformCompletionResponse) {}`

Reply:
why not just a verb? I think add a "perform" is a little bit meanless, maybe i will change it to complete

Reply:
See https://google.aip.dev/136: "The name of the method should be a verb followed by a noun."